packages/medusa/src/api/routes/store/products/list-products.ts

```diff
@@ -199,6 +199,12 @@ export default async (req, res) => {
 
   // get only published products for store endpoint
   filterableFields["status"] = ["published"]
+  // store APIs only receive active and public categories to query from
+  filterableFields["categories"] = {
+    ...(filterableFields.categories || {}),
+    is_internal: false,
```

Comment:
suggestion: I thought that we had a scope const for holding the active/available constraint, should we?

Reply:
I'm sorry, I don't understand. Can you elaborate?